customizations/mei-Neumes.xml

```diff
@@ -111,13 +111,13 @@
         <classSpec ident="model.layerPart.cmn" module="MEI.shared" type="model" mode="delete"/>
 
         <!-- Disallow events within layer -->
-        <classSpec ident="model.eventLike" module="MEI.shared" type="model" mode="replace">
+        <!-- <classSpec ident="model.eventLike" module="MEI.shared" type="model" mode="replace">
```

Comment:
it shouldn't be necessary anymore to comment lines 114 to 120. Instead, this will have side effects on editorial markup, as it effectively removes model.rdgPart.critapp.

Reply:
That is indeed correct, with the changes you suggested to allow custos and clef to appear within layer:

https://github.com/vigliensoni/music-encoding/blob/414d632919632d2f6c2dd583ff867660ce6af35d/customizations/mei-Neumes.xml#L131-L141

This is no longer required

Reply:
@kepper The change you propose actually allows to have `clef` and `custos` elements inside `layer`, which is the expected behaviour. However, it does not warn about having these elements inside a `syllable`. 

The expected behaviour should be to return a warning:

`error: element "clef" not allowed here; expected the element end-tag or element "add", "anchoredText", "annot", "app", "cb", "choice", "colLayout", "corr", "curve", "damage", "del", "div", "ending", "expansion", "gap", "handShift", "line", "orig", "pb", "reg", "relation", "relationList", "restore", "sb", "scoreDef", "section", "sic", "staff", "staffDef", "subst", "supplied" or "unclear"`

Reply:
ok, will look, but probably not before Tuesday…

Reply:
Hi @vigliensoni, apologies for the extreme delay :-( I've tried to address your issue in https://github.com/vigliensoni/music-encoding/pull/2. Could you please have another look for potential side effects? We can still get this into the schema on October 29 (assuming that today's ODD Friday will be too soon to resolve everything). Again, apologies for dropping that ball… 